lib/omnibus/licensing.rb

```diff
@@ -253,5 +297,98 @@ def local?(license)
     def licensing_warning(message)
```

Comment:
In the future, I'm hoping to make it so that this helper checks a CLI flag and fails the run if set.